### DISCUSSION TITLE
Can Prometheus be omitted?

### DISCUSSION THREAD

**Discussion by ABAG603** [grafana/mimir#9542] [Help and support] [answered]
Hi,

Considering OTEL Collector can export directly to Mimir using Prometheus Remote Write Exporter and Mimir can be selected as a datasource in Grafana Dashboard, do I still need to have a Prometheus in the system if there's no need to scrape metrics? Or in this case I can have only Mimir + Grafana Dashboard setup?

**Comment by pstibrany** [comment]
Yes, in your case you don't need Prometheus.

**Reply by 56quarters** [reply, marked as answer]
You _can_ omit it but keep in mind using Prometheus instead of an OTEL collector has some nice features like a WAL stored on disk that can be used to buffer when the remote write target (Mimir) is down for an extended period of time.
